sdk/core/core/inc/az_http_policy.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_HTTP_POLICY_H
+#define AZ_HTTP_POLICY_H
+
+#include <az_contract.h>
+#include <az_http_request.h>
+#include <az_pair.h>
+#include <az_result.h>
+#include <az_span.h>
+#include <az_str.h>
+
+#include <_az_cfg_prefix.h>
+
+// PipelinePolicies
+//   Policies are non-allocating caveat the TransportPolicy
+//   Transport policies can only allocate if the transport layer they call allocates
+// Client ->
+//  ===HttpPipelinePolicies===
+//    UniqueRequestID
+//    Retry
+//    Authentication
+//    Logging
+//    Buffer Response
+//    Distributed Tracing
+//    TransportPolicy
+//  ===Transport Layer===
+
+#define AZ_ERROR_PIPELINE_INVALID 0XF000001
```

Comment:
See az_result.h, add it there. It has macros to construct error codes and define them.
Regardless, for arbitrary integer constants, I recommend to use unnamed enums. See az_base64.c